.meteor/packages

```diff
@@ -33,7 +33,7 @@ service-configuration@1.0.11
 mdg:validated-method
 shell-server@0.3.0
 dynamic-import@0.2.0
-standard-minifier-js@2.2.3
```

Comment:
Any idea why it dropped the version for standard minifier?

Reply:
Looking into it..

Reply:
`The specified packages are at their latest compatible versions.`. Tested-docker builds still function and this was the main deal we had with the migration to this minifier.

Reply:
ok, so we're good to merge you think?

Reply:
I do 👍